src/module.ts

```diff
@@ -129,6 +133,10 @@ export default defineNuxtModule<ModuleOptions>({
 
     const resolvedOptions = nuxt.options.runtimeConfig.apiParty as Required<ModuleOptions>
 
+    Object.entries(resolvedOptions.endpoints).forEach(([id, endpoint]) => {
+      nuxt.callHook('api-party:resolve', id, endpoint)
```

Comment:
Should I add a hook after this for the entire endpoints object? Could make adding dynamic endpoints possible.

Reply:
Good question. I'd rather call it afterwards for the entire optionss object:
```ts
nuxt.callHook('api-party:resolved', resolvedOptions)
```

This would keep the hook name close to others from Nuxt like `schema:resolved` or `vite:configResolved`.

Reply:
True, I'm fine with `api-party:extend` as well.